### PR TITLE
Explain the concept of webrings

### PR DESCRIPTION
**Pull request by gabebw**
Some people might not know what a webring is, so let's explain their history, purpose, and utility.

### CODE REVIEW

File: app/views/homes/show.html.erb [hunk 1]
```diff
@@ -14,10 +14,43 @@
...
+    Do you long for a simpler time, when America was Online and the only person you
+    could Ask was Jeeves? Hotline Webring is bringing that time back, with
+    Webrings!
+  </p>
+
+  <p>
+    In the 90s, search engines sucked, and people wanted a way to discover
+    related sites. &ldquo;I wish I could see more sites about puppies,&rdquo;
+    you might say to yourself over the sonorous clang of your modem, while
+    waiting for a bitmap of a Clumber Spaniel to load.
```
Comment (edwardloveall) [inline app/views/homes/show.html.erb]: Can you link to a Clumber Spaniel?

Comment (gabebw) [inline app/views/homes/show.html.erb]: Sure, lemme find a bitmap.

File: app/views/homes/show.html.erb [hunk 2]
```diff
@@ -14,10 +14,43 @@
...
+  </p>
+
+  <p>
+    So people with websites joined webrings, which were rings of sites where
```
Comment (edwardloveall) [inline app/views/homes/show.html.erb]: s/rings/collections ?

Not sure if a "ring of sites" is clear enough?

File: app/views/homes/show.html.erb (hunk 1 above)
Comment (gabebw) [inline app/views/homes/show.html.erb]: I couldn't find a bitmap but look at this dork http://science-lakes.com/images/clumber-spaniel/clumber-spaniel8.jpg

Comment (edwardloveall) [inline app/views/homes/show.html.erb]: :+1:

**Comment by gabebw** [comment]
ok addressed all feedback

**Pull request merged**